src/nvim/mark.c

```diff
@@ -657,8 +658,18 @@ void ex_marks(exarg_T *eap)
   show_one_mark(']', arg, &curbuf->b_op_end, NULL, true);
   show_one_mark('^', arg, &curbuf->b_last_insert.mark, NULL, true);
   show_one_mark('.', arg, &curbuf->b_last_change.mark, NULL, true);
-  show_one_mark('<', arg, &curbuf->b_visual.vi_start, NULL, true);
-  show_one_mark('>', arg, &curbuf->b_visual.vi_end, NULL, true);
+
+  // Show the marks as where they will jump to.
+  startp = &curbuf->b_visual.vi_start;
+  endp = &curbuf->b_visual.vi_end;
+  if ((startp->lnum < endp->lnum || endp->lnum == 0) && startp->lnum != 0) {
```

Comment:
https://github.com/vim/vim/commit/54c3fcd852f9d986f81547429e850b3364f058d6#diff-e8d519693fa58086f6442bceaf8431c6R739
vim used the `LT_POS` macro in this line. Is the `lnum` comparison sufficient?

Reply:
Sure. I've replaced the lnum comparison with the lt() in the new commit.
I just wasn't sure which function replaced the LT_POS macro. 

(I think the lnum comparison would have been sufficient for the use case, since only the lnum is shown in :marks anyway. But using lt() would be more "in-sync" with what vim is doing.)

Question regarding the process: Should I rebase and squash the two commits together?